src/main/java/uk/gov/hmcts/reform/professionalapi/domain/entities/PaymentAccount.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.professionalapi.domain.entities;
+
+import static javax.persistence.GenerationType.AUTO;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import javax.persistence.*;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Entity(name = "payment_account")
+@NoArgsConstructor
+public class PaymentAccount {
+
+    @Id
+    @GeneratedValue(strategy = AUTO)
+    private UUID id;
+
+    @Column(name = "PBA_NUMBER")
+    private String pbaNumber;
+
+    @ManyToOne
+    @JoinColumn(name = "ORGANISATION_ID")
+    private Organisation organisation;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "USER_ID")
+    private ProfessionalUser user;
+
+    @UpdateTimestamp
+    @Column(name = "LAST_UPDATED")
+    private LocalDateTime lastUpdated;
+
+    @CreationTimestamp
+    @Column(name = "CREATED")
+    private LocalDateTime created;
+
+    public PaymentAccount(String pbaNumber) {
+        this.pbaNumber = pbaNumber;
+    }
+
+    public void setOrganisation(Organisation organisation) {
+        this.organisation = organisation;
+    }
+
+    public void setUser(ProfessionalUser user) {
```

Comment:
Multiple users can be assigned the same payment account. Most organisations only have the one account